### DISCUSSION TITLE
Serial print not working in ezWS2812

### DISCUSSION THREAD

**Discussion by yogeshmiggalore-eaton** [SiliconLabs/arduino#42] [General]
Serial print not working in ezWS2812, please check once.

**Comment by lyusupov** [comment]
Have you used `Serial.begin(115200);` first ?

**Reply by yogeshmiggalore-eaton** [reply]
Worked after following these steps
1. erase completed flash and user data using Simplicity Studio.
2. update the bootloader using arduino burn bootloader. 
3. program ezWS2812 example code with serial print.

**Comment by silabs-bozont** [comment]
Glad it works, as @lyusupov mentioned call `Serial.begin(115200);` before any printing. The _ezWS2812_ doesn't have any `Serial.print` calls by default - so it may not be initialized during it's use - if you haven't initialized it in your code before.
